privacyidea/static/components/token/controllers/tokenDetailController.js

```diff
@@ -1,4 +1,37 @@
 
+function date_object_to_string(date_obj) {
+    s = "";
+    if (date_obj) {
+        Y = date_obj.getFullYear();
+        D = date_obj.getDate();
+        D = (D>9 ? '' : '0') + D;
+        M = date_obj.getMonth()+1;
+        M = (M>9 ? '' : '0') + M;
+        h = date_obj.getHours();
+        h = (h>9 ? '' : '0') + h;
+        m = date_obj.getMinutes();
+        m = (m>9 ? '' : '0') + m;
+        o = date_obj.toTimeString().split(" ")[1];
+        // 10:20:11 GMT+0200 (CEST)
+        o = o.substring(3);
```

Comment:
Hmm, will this work on all browsers? We should find out if ``toTimeString`` is guaranteed to give the offset relative to GMT (which corresponds to UTC).
Maybe using ``getTimezoneOffset`` would be safer.